### PR TITLE
Fix `scob` when Float32 is explictly demonstrated

### LINKED ISSUE #254: Float32 on x_32 keeps it float type
In the Julia [Syntax example](https://github.com/JuliaDataScience/JuliaDataScience/blob/main/contents/julia_basics.md) where it discusses `round_number` function, 

```
x_32 = Float32(1.1)
round_number(x_32)
```
The output is incorrect as stated `1.0` as it would remain a 32bit Float, even rounded, since the round_numbers function does not reassign the type. 

The output remains at `1.0f0`

```
julia> round_number(x_32)
1.0f0
```

Thank you kindly and I could be way off base, just dipping my toes in.

### PR DESCRIPTION
**Pull request by storopoli**
This closes #254.

### CODE REVIEW

**Comment by storopoli** [comment]
@rikhuijzer see if this implementation is correct.

File: contents/julia_basics.md [hunk 1]
```diff
@@ -394,7 +394,11 @@ Now, it works as expected with any float type:
 ```jl
 s = """
     x_32 = Float32(1.1)
-    round_number(x_32)
+    x_32 = round_number(x_32)
+    io = IOBuffer(); # hide
+    show(io, r) # hide
+    x_32 = String(take!(io)) # hide
+    x_32
```
Comment (rikhuijzer) [inline contents/julia_basics.md:401]:
```suggestion
    sprint(show, x_32) # hide
```

**Comment by storopoli** [comment]
Yeah, you are a wizard!

**Comment by rikhuijzer** [comment]
Thanks for looking into it Jose!

**Pull request merged**